console/console-init/ui/src/components/MetadataListTable/TableRow.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import {
+  DataListItem,
+  DataListItemRow,
+  DataListCell,
+  DataListToggle,
+  DataListContent,
+  DataListItemCells
+} from "@patternfly/react-core";
+import classNames from "classnames";
+import { StyleSheet } from "@patternfly/react-styles";
+
+const styles = StyleSheet.create({
+  data_list_cell: {
+    marginRight: 0
+  },
+  key_cell_toogle: {
+    marginLeft: -25
+  },
+  key_cell: {
+    marginLeft: 50
+  },
+  type_cell_toggle: {
+    textAlign: "center",
+    textTransform: "capitalize"
+  },
+  type_cell: {
+    textAlign: "center",
+    marginLeft: 55,
+    textTransform: "capitalize"
+  },
+  value_cell: {
+    textAlign: "center"
+  }
+});
+
+export type ITableRowProps = {
+  rowData: any;
+};
+
+export enum DataType {
+  ARRAY = "array",
+  OBJECT = "object"
+}
+
+export const TableRow: React.FC<ITableRowProps> = ({ rowData }) => {
+  const [expanded, setExpanded] = useState<string[]>([]);
```

Comment:
we can modify the name of expanded as expandedList as its a bit confusing

Reply:
done